pkg/github/github.go

```diff
@@ -40,10 +40,34 @@ type Config struct {
 	maxRetryDelay     time.Duration
 }
 
+// Issue is the GitHub Issue.
+type Issue struct {
+	Number *int
+}
+
+type IssueComment struct {
+	ID *int64
```

Comment:
Why make these pointers? Do we expect nil ints?

Reply:
Updated.